assets/packages/limesurvey/survey.js

```diff
@@ -66,7 +66,7 @@ function triggerEmRelevanceSubQuestion(){
                     $(item).prop("disabled", false );
                 });
             } else {
-                $(event.target).find('.answer-item input').each(function(itrt, item ){
+                $(event.target).find('.answer-item input, input.button-item').each(function(itrt, item ){
```

Comment:
My opinion : twig file must be updated, it's an issue in theme.

Else : next time we add `input.my_item, .mybroken-theme-item .tag`

Can you try adding answer-item class at https://github.com/LimeSurvey/LimeSurvey/blob/59d73eeb8067ef5f916341274c670eda33aaa5fd/themes/question/bootstrap_buttons_multi/survey/questions/answer/multiplechoice/rows/answer_row.twig#L24 ?